Cargo.toml

```diff
@@ -20,6 +20,7 @@ appveyor = { repository = "rust-lang-libs/libc" }
 [features]
 default = ["use_std"]
 use_std = []
+stdbuild = []
```

Comment:
Can this be avoided in this Cargo.toml? Ideally this isn't exported as part of the public API